applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -393,7 +413,53 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
     ///@}
     ///@name Protected operators
     ///@{
+    
+    void BuildRHSNoDirichlet(
+        ModelPart& rModelPart,
+        TSystemVectorType& rb)
+    {
+        KRATOS_TRY
 
+        // Get ProcessInfo from main model part
+        const auto& r_current_process_info = rModelPart.GetProcessInfo();
+
+        auto& r_elements = mHromSimulation ? mSelectedElements : rModelPart.Elements();
+        if(!r_elements.empty())
+        {
+            block_for_each(r_elements, Kratos::Vector(), [&](Element& r_element, Kratos::Vector& rhs_elem)
+            {
+                DofsVectorType dofs = {};

```

Comment:
```suggestion
                DofsVectorType dofs;

```
I think you don't need the initializing braces.